optuna_dashboard/ts/components/ThreejsArtifactViewer.tsx

```diff
@@ -4,14 +4,17 @@ import { Canvas } from "@react-three/fiber"
 import { GizmoHelper, GizmoViewport, OrbitControls } from "@react-three/drei"
 import { STLLoader } from "three/examples/jsm/loaders/STLLoader"
 import { Rhino3dmLoader } from "three/examples/jsm/loaders/3DMLoader"
+import { OBJLoader } from "three/examples/jsm/loaders/OBJLoader"
 import { PerspectiveCamera } from "three"
 import { Modal, Box, useTheme } from "@mui/material"
 import ClearIcon from "@mui/icons-material/Clear"
 import IconButton from "@mui/material/IconButton"
 
 export const isThreejsArtifact = (artifact: Artifact): boolean => {
   return (
-    artifact.filename.endsWith(".stl") || artifact.filename.endsWith(".3dm")
+    artifact.filename.endsWith(".stl") ||
+    artifact.filename.endsWith(".3dm") ||
+    artifact.filename.endsWith(".obj")
```

Comment:
Since the obj extension is also present in files other than 3D models, should I make mimetype a criterion as well?